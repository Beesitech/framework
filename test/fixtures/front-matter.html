---
foo: "bar"
---

<div>foo is: {{ page.foo }}</div>
