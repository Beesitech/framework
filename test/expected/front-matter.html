<div>foo is: bar</div>
